flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/WordCountTable.java

```diff
@@ -48,9 +50,9 @@ public static void main(String[] args) throws Exception {
 		Table table = tEnv.fromDataSet(input);
```

Comment:
Update `org.apache.flink.table.examples.java.WordCountSQL` and  classes in `org.apache.flink.table.examples.scala` (with `$""`) as well.